packages/react-ui/src/theme/index.ts

```diff
@@ -92,4 +125,19 @@ const spacing = {
     10: "192px",
 };
 
-export { colors, fontWeights, fontSizes, lineHeights, borderRadius, dropShadow, spacing };
+const useTheme = () => {
+    const theme = useEmotionTheme();
```

Comment:
not sure how themes with the `css` prop will work in cases where there's no emotion provider
Maybe we can just avoid using it

```typescript
<Thing css={(theme) => ???}/>
```